packages/fluentui/accessibility/src/behaviors/Tree/treeItemBehavior.ts

```diff
@@ -66,9 +68,10 @@ const treeItemBehavior: Accessibility<TreeItemBehaviorProps> = props => ({
       },
       ...(props.selectable && {
         performClick: {
-          keyCombinations: props.hasSubtree
-            ? [{ keyCode: keyboardKey.Spacebar }, { keyCode: keyboardKey.Enter }]
-            : [{ keyCode: keyboardKey.Spacebar }],
+          keyCombinations: props.hasSubtree ? [{ keyCode: keyboardKey.Enter }] : [{ keyCode: keyboardKey.Spacebar }],
+        },
+        performSelection: {
```

Comment:
Should we add condition here on when we add this action handler?

Reply:
It's already inside the condition `...(props.selectable && {`

Reply:
Got it! I missed that